CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this packages will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
You can remove this, we no longer manually update the changelog. If a test was failing on this you may need to merge main?